includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -566,6 +538,43 @@ private function getCommunity() {
 		return $data;
 	}
 
+	private function getAdvertise() {
+		$data = [
+			'header' => [
+				'type' => 'line-text',
+				'title' => [
+					'type' => 'translatable-text',
+					'key' => 'global-footer-advertise-header'
+				]
+			],
+			'links' => [
+				[
+					'type' => 'link-text',
+					'title' => [
+						'type' => 'translatable-text',
+						'key' => 'global-footer-advertise-link-media-kit'
+					],
+					'href' => $this->getHref( 'media-kit' ),
+					'tracking_label' => 'advertise.media-kit',
+				]
+			]
+		];
+
+		if ( $this->getHref( 'media-kit-contact' ) ) {
+			$data['links'][] = [
+				'type' => 'link-text',
+				'title' => [
+					'type' => 'translatable-text',
+					'key' => 'global-footer-advertise-link-contact'
+				],
+				'href' => $this->getHref( 'media-kit-contact' ),
+				'tracking_label' => 'advertise.contact',
+			];
+		}
+
+		return $data;
+	}
+
 	private function getLocalSitemapUrl() {
 		if ( $this->product !== static::PRODUCT_FANDOMS ) {
 			$default = true; // $wgEnableLocalSitemapPageExt = true; in CommonSettings
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)